Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/TracingProgress.kt

```diff
@@ -2,9 +2,15 @@ package de.rki.coronawarnapp.tracing
 
 sealed class TracingProgress {
 
-    object Idle : TracingProgress()
+    object Idle : TracingProgress() {
+        override fun toString(): String = "TracingProgress.Idle"
```

Comment:
is this necessary?

Reply:
Not necessary, but nice, otherwise it prints to debug log like `abcdefasdas.asd.asda.sd@asdasdkasdk.TracingProgress.Idle`.